### ISSUE TITLE
date_format doesn't support time formatting

### ISSUE THREAD

**Issue by lorin** [has2k1/mizani#7]
date_format returns zeros for `%H`, `%M` and `%S`

```
>>> from datetime import datetime
>>> import time
>>> from mizani import date_format

>>> fmt = "%Y-%m-%d %H:%M:%S"
>>> ts = datetime(2017,12,1,16,5,7)

>>> ts.strftime(fmt)
'2017-12-01 16:05:07'
>>> date_format(fmt)([ts])
['2017-12-01 00:00:00']
```

I looked to see if there was a different formatter for datetimes instead of just dates, but I couldn't find one.

**Comment by has2k1** [comment]
Fix released in v0.4.3.

**Comment by lorin** [comment]
Wow, that was fast!
